ansible/roles/stack-sunbird/templates/dhiti-service.env

```diff
@@ -0,0 +1,85 @@
+APPLICATION_PORT_NUMBER={{dhiti_application_port_number}}
+APPLICATION_HOST_NAME={{dhiti_application_host_name}}
+APPLICATION_BASE_URL={{dhiti_application_base_url}}
+NODE_ENV={{dhiti_node_env}}
```

Comment:
Can we use {{ env }} value here from all.yml? Like dev, staging etc?

Reply:
Sunbird uses {{env}} 
for ex: dev

This App uses development.